src/core/inc/blocks/CntSensorConfigBlock.h

```diff
@@ -0,0 +1,26 @@
+#ifndef TIN_CNTSENSORCONFIGBLOCK_H
+#define TIN_CNTSENSORCONFIGBLOCK_H
+
+
+#include <vector>
+#include <netinet/in.h>
+#include "AbstractBlock.h"
+
+class CntSensorConfigBlock : public AbstractBlock {
+public:
+    std::vector<std::string> central_ips;
+    in_port_t port_id;
+    std::string cnt_ip;
+
+    CntSensorConfigBlock() : AbstractBlock(bt_cnt_sensor_config) {}
```

Comment:
bt_xxx -> BlockType: xxx

CntSensorConfig -> tak było gdzieś w dokumentacji, cnt to chyba centralka